README.md

```diff
@@ -79,4 +79,27 @@ docker service logs -f smart_postgres
 ```
 
 NOTES:
-to trigger a new image build an push with circle, just create a new tag or delete and recreate an existing tag.
\ No newline at end of file
+to trigger a new image build an push with circle, just create a new tag or delete and recreate an existing tag.
+
+
+## Configuring SMART Desktop to access the Server
+
+Additional steps must now be taken in order for a SMART Desktop installation to be able to access the server.
+
+The certificate to be used when configuring SMART Desktop to access the server is found at https://github.com/arribada/SMARTConnect/blob/master/tomcat/ssl.cert
+
+Additionally, in order to use this certificate you must add an alias to your host file so that the IP address of the hosting machine is usable with the certificate.
+
+First go to https://www.whatismyip.com/ on the machine hosting the Connect Server to find the IP address of that machine.
```

Comment:
```suggestion
```
let's not put specific network details as these could vary quite a bit, for example, can access the smart connect server locally, than using the local IP would be preferred, over the internet would need port forwarding, with VPN would need another setup etc. Lets leave out these details as they vary quite a bit.